multimedia/libvidstab/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        georgmartius vid.stab 1.1.0 v
+
```

Comment:
please add `github.tarball_from releases` (if upstream provides those) or, alternatively `github.tarball_from archive`. You will need to recalculate the checksums, after cleaning the port locally with `sudo port clean --all libvidstab`